tooling/build/scripts/generate-sitemap.js

```diff
@@ -244,10 +244,27 @@ const getSiteMapChildrenEntries = async (fullPath, relativePath) => {
 
   children.push(...danglingDirEntries)
 
-  // Ensure that the result is ordered in alphabetical order
-  children.sort((a, b) =>
-    a.title.localeCompare(b.title, undefined, { numeric: true }),
-  )
+  children.sort((a, b) => {
+    const aPermalink = a.permalink.split("/").pop()
+    const bPermalink = b.permalink.split("/").pop()
+
+    if (!pageOrderData) {
+      return a.title.localeCompare(b.title, undefined, { numeric: true })
+    }
```

Comment:
saw that publishing script also have this comparison
```ts
pageOrder.indexOf(aPermalink) === pageOrder.indexOf(bPermalink)
```

just want to confirm it's not needed here?

Reply:
Yup it's not required, cos the way the `pageOrder` and `pageOrderData` is initialised is slightly different, so here the check is not needed.